src/components/App.tsx

```diff
@@ -42,14 +47,58 @@ const App: FC<AppProps> = ({ demoId }) => {
     setEvaluatePath(true);
   }, []);
 
+  const setMcodeRecords = useCallback((resources: DomainResource[]): void => {
+    // Create a Bundle for the CQL engine
```

Comment:
We do this in so many places I wonder if it makes more sense to just make the ELM executor take in an array instead of a bundle

Reply:
https://jira.mitre.org/browse/PATHWAYS-242